spectrum_utils/spectrum.py

```diff
@@ -1037,23 +996,25 @@ def annotate_peptide_fragments(self, fragment_tol_mass: float,
             raise ValueError('No peptide sequence available for the spectrum')
         if max_ion_charge is None:
             max_ion_charge = max(1, self.precursor_charge - 1)
-
+        if neutral_losses is None:
+            neutral_losses = {None: 0}
```

Comment:
I think this is the thing that matters to me in the future proofing of the neutral losses. Right now a dict of masses can be specified. In the future, (not now), if a dict is passed in where its "str"->"str" that could mean new symbol pointing to formula, but if its "str"->"float" that could mean symbol to a pre-calculated mass loss/gain.